python/tvm/auto_scheduler/measure.py

```diff
@@ -1079,7 +1079,7 @@ def _timed_rpc_run(
                 if arg in tensor_input_map:
                     tensor_name = tensor_input_map[arg]
                     if tensor_name in task_input_names:
-                        args.append(get_task_input_buffer(inp.task.workload_key, tensor_name))
+                        args.append(ndarray.array(get_task_input_buffer(inp.task.workload_key, tensor_name), ctx))
```

Comment:
Thanks!
Let's add an extra UT to `tests/python/unittest/test_auto_scheduler_measure.py`, copy the `test_measure_special_inputs_map_by_name()` and modifiy the runner to a `LocalRPCMeasureContext().runner`.